p2plending/frontend/src/app/titles/TitlesResults.js

```diff
@@ -76,7 +65,7 @@ class TitlesResults extends Component {
   };
 
   componentDidUpdate(prevProps) {
-    if (!this.hasSearchTermInUrlChanged(prevProps.match, this.props.match)) {
+    if (!hasParamInUrlChanged(prevProps.match, this.props.match, "searchTerm")) {
```

Comment:
Ahhh, I see what you did here, nice. I think there might be a better way to do this function rather than passing in this third parameter, "searchTerm". I don't know how to do it but I may look at replacing this in the future. Good creativity though